.github/workflows/deploy-changed-cf.yaml

```diff
@@ -33,13 +33,12 @@ jobs:
         name: Check CloudFormation templates
         run: |-
           # Merge all modified and renamed files into one JSON array
-          #jq -s add ${HOME}/files_added.json ${HOME}/files_modified.json ${HOME}/files_renamed.json > ${HOME}/files_to_check.json
+          jq -s 'add | unique' ${HOME}/files_added.json ${HOME}/files_modified.json ${HOME}/files_renamed.json > ${HOME}/files_to_check.json
```

Comment:
Without `unique` I would have repeated tempaltes, liek for example rds.yaml

Reply:
good find!